app/controllers/base_recipes.py

```diff
@@ -54,8 +54,8 @@ def calcRecipeAJAX(self):
 
         try:
             result = calculations.calculate_recipe(ingredients, diet)
-        except Exception as e:
-            abort(400, str(e.args[0]))
+        except ValueError:
```

Comment:
400 je error -> posílalo se to do Sentry. Přitom jde jen o nemožný výpočet, proto takhle lepší.